cwt/draft-ietf-ace-cbor-web-token.xml

```diff
@@ -912,7 +912,7 @@ a70175636f61703a2f2f61732e6578616d706c652e636f6d02656572696b7703
 
 	<section anchor="exampleKeysSymmetric128" title='128-bit Symmetric Key as Hex Encoded String'>
 	  <figure><artwork><![CDATA[
-8e82e68e61654ecb5a369fe8be7572dd
+8e82e68e61654ecb5a369fe8be7572
```

Comment:
It seems like there was an unintentional deletion of two characters above, as the new value is only 120 bits.